scraper/src/github-scraper/discussion.ts

```diff
@@ -52,43 +44,48 @@ async function fetchDiscussionsForOrg(org: string, cursor = null) {
   const response = await octokit.graphql.paginate(query, variables);
 
   type Edge = typeof response.organization.repositories.edges;
-  // const discussions = response.organization.repositories.edges.map(
-  //   (edge: Edge) => edge.node.discussions.edges,
-  // );
 
-  // return discussions.flat();
   const discussions = response.organization.repositories.edges.map(
     (edge: Edge) => ({
       repoName: edge.node.name,
       discussions: edge.node.discussions.edges,
     }),
   );
+
   return discussions;
 }
 
 async function parseDiscussionData(
   allDiscussions: { repoName: string; discussions: Discussion[] }[],
+  date: string,
+  numDays: number,
 ) {
-  const parsedDiscussions: ParsedDiscussion[] = allDiscussions.flatMap((repo) =>
-    repo.discussions.map((d) => {
-      const participants = Array.from(
-        new Set(d.node.comments.edges.map((c) => c.node.author.login)),
-      );
-      return {
-        source: "github",
-        title: d.node.title,
-        text: d.node.body,
-        author: d.node.author.login,
-        link: d.node.url,
-        time: d.node.createdAt,
-        category: {
-          name: d.node.category.name,
-          emoji: d.node.category.emojiHTML.replace(/<\/?div>/g, ""),
-        },
-        participants,
-        repoName: repo.repoName,
-      };
-    }),
+  const endDate: Date = startOfDay(parseISO(date));
+  const startDate: Date = startOfDay(subDays(endDate, numDays));
```

Comment:
this is duplicated twice in the scraper. Pass the date range objects directly instead of repeating the logic. Easier to maintain